Gemfile

```diff
@@ -56,3 +58,9 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 group :production do
   gem 'pg', '0.18.4'
 end
+
+# replaces npm uglifier which does not work on heroku
+gem 'uglifier'
```

Comment:
donno, I ran into the issue and did the most recent fix [here](http://stackoverflow.com/questions/27659703/heroku-build-fails-on-uglifier)